environment.yml

```diff
@@ -0,0 +1,13 @@
+name: gene-ontology
+channels:
+  - conda-forge
+dependencies:
+  - conda-forge::networkx=2.2
+  - conda-forge::numpy=1.15.3
+  - conda-forge::pandas=0.23.4
+  - conda-forge::python=3.6.7
+  - pip:
+    - jupyter==1.0.0
```

Comment:
I am not sure exactly what this package/bundle is. As far as I can tell it is [always](https://pypi.org/project/jupyter/#history) v1.0.0. Instead, I think we want to install `notebook`, `nbconvert`, and `ipykernel` which seem to follow actual semantic versioning and get updated. Also these can all be installed from conda-forge.